### DISCUSSION TITLE
[FEAT] Why there is only frontend code generated?

### DISCUSSION THREAD

**Discussion by iam-frankqiu** [refinedev/refine#4711] [General]
### Is your feature request related to a problem? Please describe.

I chose  NestJS and other frontend frameworks. When I exported the code. I found there only have NextJS code was downloaded. I  was wondering whether I can export the NestJS code. 

### Describe the thing to improve

I hope we can export both frontend and backend code.

**Comment by alicanerdurmaz** [comment]
Hello @iam-frankqiu, 

When you choose NestJS, the `create refine-app` will bootstrap your project with [nestjsx-crud](https://github.com/refinedev/refine/tree/master/packages/nestjsx-crud).
With the `nestjsx-crud` data provider, you can easily connect to an API built using [NestJS](https://nestjs.com/) and [nextjs-crud](https://github.com/nestjsx/crud).

**refine** is a frontend framework based on React, and it remains agnostic regarding the backend. We offer 15+ `dataProvider` options to easily connect to your API, and `nestjsx-crud` is one of them.

In summary, **refine** currently doesn't generate any backend code.

For more information, please refer to the [Data Provider documentation. →](https://refine.dev/docs/tutorial/understanding-dataprovider/index)
